docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -58,26 +52,34 @@ code: |
   process_people_variables
   ask_people_quantity_question  
   add_includes
-  set_metadata # Adds metadata block to the interview as first block
+  add_metadata
+  add_short_title
+  add_form_type  
   choose_field_types
   questions.gather()
   show_screen_order
   add_objects_block
   add_sections  
-  add_logic_tree
+  add_interview_order
   add_draft_intro_screen
   add_questions_to_interview
-  created_preview_question  
+  add_preview_question  
   add_review_screen  
   # Insert comment block
-  add_short_title
-  add_form_type
-  created_attachment
+  add_end_question
+  add_attachment
   wrote_interview
   increment_sequence
 
   show_interview
 ---
+# Temporary to text new Mako style questions
```

Comment:
Is this `Temporary to test new Mako style questions`? If it is a test block, should remove.